.github/workflows/create-or-update-release-pr.yml

```diff
@@ -0,0 +1,59 @@
+name: Create or Update Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: Release type
+        options:
+          - patch
+          - minor
+      release_scope:
+        type: choice
+        description: Release Scope
+        options:
+          - experimental
+          - sdk
+          - all
+
+jobs:
+  create-or-update-release-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          repository: opentelemetrybot/opentelemetry-js
+          ref: main
+          token: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
+      - name: Sync with upstream
+        run: |
+          git remote show origin
+          git remote add upstream https://github.com/open-telemetry/opentelemetry-js.git
+          git fetch upstream
+          git reset --hard upstream/main
+          git push origin main --force
+
+      - uses: actions/setup-node@v4
+        with:
+          cache: 'npm'
+          cache-dependency-path: |
+            package-lock.json
+          node-version: 22
+      - run: npm install -g npm@latest
```

Comment:
nit: Personally I'm a little hesistant to always jump to the latest npm release. This could be a new major version of npm. It makes this workflow slightly less reproducible. 
The npm included with the latest Node.js 22 should suffice.

Arguably we have the same moving target by getting whatever the latest Node.js "22" release is already, so this is a minor complaint.

Reply:
Sounds good. Thanks.